features/upload_cita_appointments.feature

```diff
@@ -0,0 +1,26 @@
+Feature: Manual Upload of CITA appointments CSV
+  As a Pension Wise data analyst
+  I want to be able to manually upload the CITA appointments CSV
+  So to process the generated data file
```

Comment:
This doesn't make much sense, also it doesn't really represent a value proposition for the analyst. Why do they care that they can upload the file?